tests/by-util/test_pidof.rs

```diff
@@ -33,3 +33,29 @@ fn test_no_program() {
 fn test_no_pid_found() {
     new_ucmd!().arg("NO_THIS_PROGRAM").fails().code_is(1);
 }
+
+#[test]
+#[cfg(target_os = "linux")]
+fn test_quiet() {
+    let binding = new_ucmd!().arg("kthreadd").arg("-q").succeeds();
+    let output = binding.stdout();
+
+    assert!(output.is_empty())
```

Comment:
You can simplify it to:
```suggestion
    new_ucmd!().arg("kthreadd").arg("-q").succeeds().no_output();
```